Uni_life_tracker/app/src/main/java/com/generic/plannr/MainPageActivity.java

```diff
@@ -6,7 +6,7 @@
 import android.app.AlertDialog;
 import android.content.Intent;
 import android.view.View;
-import android.widget.TextView;
+import android.widget.*;
```

Comment:
do we need to import all widgets, can we just import the ones we need?

Reply:
we would have to make 5 imports